.circleci/config.yml

```diff
@@ -186,32 +182,46 @@ jobs:
           k8s_svc_name: "charts-binary-com"
       
 workflows:
-  release_dev:
+  build_and_push_dev:
     jobs:
-      - build_push_dev:
+      - test:
           filters:
             branches:
-              only: /^dev$/
-          context: binary-frontend-artifact-upload
+              only: 
+                - /^dev/
+                - /pull\/.*/
```

Comment:
we can change this to allow all the branches. i.e; it needs to be triggered for merges, PRs, and releases